xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearchMacros.xml

```diff
@@ -67,7 +67,10 @@
     &lt;/div&gt;
     &lt;div class="col-xs-12 col-sm-6"&gt;
       &lt;div class="input-group"&gt;
-        &lt;input type="search" name="text" class="form-control withTip useTitleAsTip"
+        &lt;label class="sr-only" for='search-page-bar-input'&gt;
+        $services.localization.render('search.page.bar.query.title')
+        &lt;/label&gt;
+        &lt;input id='search-page-bar-input' type="search" name="text" class="form-control withTip useTitleAsTip"
```

Comment:
Same here, it seems inconsistent to use `'` and `"` for different attributes of the same element.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2187/commits/013015e347f52b417dc472e6eef7802c2e25d65e